validation/custom_constraint.rst

```diff
@@ -250,3 +250,73 @@ not to the property:
                 $metadata->addConstraint(new ProtocolClass());
             }
         }
+
+How to Unit Test your Validator
+-------------------------------
+
+To create a unit test for you custom validator, you can use ``ConstraintValidatorTestCase`` class. All you need is to extend
```

Comment:
Maybe we can rewrite and shorten the paragraph a bit like this:

> To create a unit test for you custom validator your test class should extend the `ConstraintValidatorTestCase` class and implement the `createValidator()` method:

Reply:
agree